Rdmp.Core/Curation/Data/DataLoad/LoadMetadataCatalogueLinkage.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) The University of Dundee 2024-2024
+// This file is part of the Research Data Management Platform (RDMP).
+// RDMP is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+// RDMP is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+// You should have received a copy of the GNU General Public License along with RDMP. If not, see <https://www.gnu.org/licenses/>.using Amazon.Auth.AccessControlPolicy;
+
+using Rdmp.Core.Curation.Data.ImportExport;
+using Rdmp.Core.Curation.Data.Serialization;
+using Rdmp.Core.Repositories;
+using Rdmp.Core.ReusableLibraryCode;
+using System;
+using System.Collections.Generic;
+using System.Data.Common;
+using System.Diagnostics.CodeAnalysis;
```

Comment:
@JFriel We have a conflict here with the `NotNull` declared in Rdmp.Core/ReusableLibraryCode/Annotations/Annotations.cs - probably want to rename that one at some point soon? (Mainly flagging that for later discussion, rather than this PR.)